concepticondata/conceptlists.tsv

```diff
@@ -412,4 +412,5 @@ RonaTas-2011-431	Róna-Tas, András and Berta, Árpád	2011		431	historical	Engl
 Swadesh-1961-100	Swadesh, Morris	1961		100	basic	Spanish	Mexican Languages		Swadesh1961		This concept list is Swadesh's translation of the [Swadesh list of 100 items](:ref:Swadesh-1955-100) to Spanish.	242-266	
 Ferre-2022-160	Ferré, Pilar and Guasch, Marc and Stadthagen-Gonzalez, Hans and Comesaña, Montserrat	2022		160	ratings	English	English, Portuguese	https://figshare.com/ndownloader/files/34091276	Ferre2022		This list features affective ratings for a set of emotional and neutral English words (nouns and adjectives) from a group of English native speakers and a group of European Portuguese–English bilinguals.		
 Gerardi-2022-705	Gerardi, Fabrício and Aragon, Carolina Coelho and Reichert, Stanislav	2022		705		English	Katukina, Arawa, Harakbmut	https://github.com/lexibank/kahd	Gerardi2022		This conceptlist was collected for a database of three Amazonian languages, Katukina, Arawa, and Harakmbut. The list was constantly extended, which is why the list given here shows more items than the authors mention in their original database paper.		
-Simko-2020-205	Šimko, Ivan and Kelih, Emmerich	2020		205	basic	English	Bulgarian		Simko2020		This concept list is a translation of the [Swadesh list of 200 items](:ref:Swadesh-1952-200), extended by 5 (according to the authors 12) terms, into Modern Bulgarian.	229-243	
\ No newline at end of file
+Simko-2020-205	Šimko, Ivan and Kelih, Emmerich	2020		205	basic	English	Bulgarian		Simko2020		This concept list is a translation of the [Swadesh list of 200 items](:ref:Swadesh-1952-200), extended by 5 (according to the authors 12) terms, into Modern Bulgarian.	229-243	
+Wilson-2015-173	Wilson, Fiona M.	2015		173	basic	Latin	Spanish		Wilson2015		This concept list is a reduced version of the [Swadesh list of 200 items](:ref:Swadesh-1952-200) in Latin and Spanish.	1-73
```

Comment:
One thing I just noticed is the tag for the list. Could you please add the tags `ranked` and `norms`? Also, could you add a sentence that the list includes a ranking and frequency for Modern Spanish.